### PR TITLE
Add build

### PR DESCRIPTION
**Pull request by lukejpreston**
You can now install this manually very easily

### CODE REVIEW

**Comment by shaunbent** [comment]
Hey Luke,

Thanks for you contribution.

I've got to be 100% honest if you, I'm not really sure what we get from this change. We already had a build process within an `npm` script which appears to do the same as what your `node` build script does.

As you pointed #21 the compiled version was out of date which was causing your issue. I think the problem here is that we're not forcing the bundled compiled version to be built automatically when new versions are pushed.

**Comment by lukejpreston** [comment]
I am not using sass I am using post-css. I have to copy the dist output in order to use the css code with post-css. So the following problems occur:

From a user point of view:
1. Location changes of node_modules across major node versions (happened twice I think) means I can not reliable copy the dist file when I choose to upgrade my node version
2. I can not rely on your dist folder, or you have to have a complex build to make sure any changes are duplicated and you have nothing to catch you when this doesn't happen/work
3. I can not build it myself without checking the project out, which is really complicated and if I do not use sass it will take even longer for me to understand how that works should I want to build it manually

This script fixes all those things without the user having to think about how your project works or is built

I also would like to point out; with this in you do not need a dist folder at all and can remove the dist task. But if you remove the dist folder and dist task I guess that is a major version change which doesn't seem needed right now.

**Comment by matt-crazypanda** [comment]
@lukejpreston I don't understand why this isn't being done in your own project? This is a sass library, what what happens if you are using a project with PHP/Ruby/Java, the nodejs stuff then becomes redundant.

**Comment by lukejpreston** [comment]
You guys have provided a dist folder and you aren't maintaining it.
That is why I am building it.
If you don't want people to use the dist folder do not commit it and do not tell people to use it.

This is just making the manual installation less manual. 

This is you manual installation currently:
```
git clone https://github.com/bbc/gel-grid
cd gel-grid
npm i
npm run dist
cp dist/gel-grid.css dist/gel-grid.min.css <destination>
```

This takes ages to run and automating it is no longer trivial.

The manual installation I have added in the readme takes a fraction of the time and is less prone to errors and if you guys decide to change the project the installation step should be kept the same or very similar. Also if someone is not using node and they do not want to use sass they are going to have to use node. They need to run your dist task.

**Comment by matt-crazypanda** [comment]
@lukejpreston if its only for creating the dist release, then a job can be created on CI which builds the dist folder to keep it upto date, or we can add a git push hook for contributors to use. All I am trying to say is, if i am using a project in another language, without npm/node I probably don't want to install it just to generate a css version.

**Comment by lukejpreston** [comment]
Problems:
1. You have to maintain a CI
2. You still have to maintain the dist folder
3. You do not need a hook in order to contribute, hence on any change if there is no new dist folder you have to ask whether there should be one.

You should just remove the dist folder and the manual installation step from the README. Would be loads quicker and more reliable.

**Comment by lukejpreston** [comment]
It is a small, non-intrusive change. I don't understand the resistance.

File: src/gel-grid.scss [hunk 1]
```diff
@@ -3,8 +3,8 @@
 //\*------------------------------------*/
 
 // Include general dependancies
-@import '../node_modules/gel-sass-tools/sass-tools';
```
Comment (lukejpreston) [inline src/gel-grid.scss]: Would like to point out that this forces people to have to use node, they should be setting gel-sass-tools and sass-mq in their includes path when using sass as I have done in your script

Comment (crouchygt) [inline src/gel-grid.scss]: It doesn't, outside of a node project I would use the compiled version

Comment (lukejpreston) [inline src/gel-grid.scss]: don't care give me an update dist 😉

**Pull request closed without merging**